stories/.storybook/main.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  stories: ['../src/*.stories.tsx'],
+  addons: ['@storybook/addon-knobs'],
+}
```

Comment:
[see this](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#to-mainjs-configuration) for an explanation of this churn